tempto-core/src/main/java/com/teradata/tempto/internal/hadoop/FileSystemDataSourceWriter.java

```diff
@@ -12,51 +12,49 @@
  * limitations under the License.
  */
 
-package com.teradata.tempto.internal.hadoop.hdfs;
+package com.teradata.tempto.internal.hadoop;
 
 import com.google.common.base.Stopwatch;
+import com.google.inject.Inject;
 import com.teradata.tempto.fulfillment.table.hive.HiveDataSource;
-import com.teradata.tempto.hadoop.hdfs.HdfsClient;
-import com.teradata.tempto.hadoop.hdfs.HdfsClient.RepeatableContentProducer;
-import com.teradata.tempto.internal.hadoop.hdfs.revisions.RevisionStorage;
+import com.teradata.tempto.hadoop.FileSystemClient;
+import com.teradata.tempto.hadoop.FileSystemClient.RepeatableContentProducer;
+import com.teradata.tempto.internal.hadoop.revisions.RevisionStorage;
 import org.slf4j.Logger;
 
-import javax.inject.Inject;
-
 import java.util.Optional;
 import java.util.concurrent.TimeUnit;
 
 import static org.slf4j.LoggerFactory.getLogger;
 
-public class DefaultHdfsDataSourceWriter
-        implements HdfsDataSourceWriter
+public class FileSystemDataSourceWriter
 {
+    private static final Logger LOGGER = getLogger(FileSystemDataSourceWriter.class);
 
-    private static final Logger LOGGER = getLogger(DefaultHdfsDataSourceWriter.class);
-
-    private final HdfsClient hdfsClient;
+    private final FileSystemClient fsClient;
     private final RevisionStorage revisionStorage;
 
     @Inject
-    public DefaultHdfsDataSourceWriter(HdfsClient hdfsClient,
-            RevisionStorage revisionStorage)
+    public FileSystemDataSourceWriter(FileSystemClient fsClient,
+                                      RevisionStorage revisionStorage)
```

Comment:
undo or make it a separate (refactor) commit. Moreover I think we should keep with Presto convention, then it should look like:
```
public Constructor(
    Parameter paramenter,
    OtherParameter otherParameter)
{
```